site-user-management/layouts/api_user_deletion.tpl

```diff
@@ -172,7 +170,23 @@
           }
         }
 
-        const getUsers = (pageNumber = 1) => new Promise((resolve, reject) => {
+        const getAllUsers = () => {
+          let promises = [];
+
+          getUsersPage().then((obj) => {
```

Comment:
`obj` on hästi halb muutuja nimi, sest ei kirjelda midagi. Parem nt `usersPage`. (Märkuse korras: sellele reeglile on erandeid, nt väga lühikeses abifunktsioonis, või väljakujunenud kombed (`for` tsüklimuutuja on `i` hiljemalt 70-ndatest, [`reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) callback-funktsioonis kasutan enamasti `acc` ja `cur` jne))